### PR TITLE
WEEK-2/ASSIGMENTS

### PR DESCRIPTION
**Pull request by surajnikhade**

### CODE REVIEW

File: Week-2/Assignment-1/favourite-musician.html
Comment (NishantNaithani) [inline Week-2/Assignment-1/favourite-musician.html:1]: Here, `<footer></footer>` should be used for the footer related things.

File: Week-2/Assignment-1/line-up.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          <p>
+            <select name="gameDate">
+              <option value="March 20">March 20</option>
+              <option value="March 24">March 24</option>
+              <option value="April 2">April 2</option>
+              <option value="April 6">April 6</option>
+              <option value="April 13">April 13</option>
+            </select>
+          </p>
+        </fieldset>
```
Comment (NishantNaithani) [inline Week-2/Assignment-1/line-up.html]: Here, we don't need to wrap the `Game Date` component inside `<fieldset>...</fieldset>`

File: Week-2/Assignment-2/auto-loan.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="auto-loan.css" />
+    <title>Loan Offers</title>
+  </head>
+  <body>
+    <h1>Comparison of Dealer Incentives and Loan Offers</h1>
+
+    <div class="container">
```
Comment (NishantNaithani) [inline Week-2/Assignment-2/auto-loan.html]: Here,
1. Please wrap all your content in a parent tag.
2. The `Heading` and the `content` is not sync with each other.

File: Week-2/Assignment-2/news-article.html
Comment (NishantNaithani) [inline Week-2/Assignment-2/news-article.html:1]: Here, wrap your main content under a parent tag.

File: Week-2/Assignment-2/sleep-reccomendation.html
Comment (NishantNaithani) [inline Week-2/Assignment-2/sleep-reccomendation.html:1]: Here, `source` content is missing.

**Review by NishantNaithani — CHANGES_REQUESTED** [review]

**Comment by NishantNaithani** [comment]
@surajnikhade ,
1. I believe the `Advanced CSS` assignment is missing.
2. CSS file indentations needs to be fixed.

Thanks

**Review by surajnikhade — COMMENTED** [review]
I have pushed week-2 assignement -3  also resolved pervious comments . Please check and LMK if changes required.
@NishantNaithani  
Thanks!

**Review by NishantNaithani — APPROVED** [review]

**Pull request merged**